### ISSUE TITLE
using influx as metrics_plugin_name value doesn't work

### ISSUE THREAD

**Issue by tyge68** [Zooz/predator#202]
**Describe the bug**
Based on the documentation , I have set the metrics_plugin_name to use influx, and set the influx_metrics with the properties (host, username, password, database), when running a test, nothing happen. The predator-runner end with undefined error 

```
2019-08-19T22:31:14.740470889Z /usr/app/utils/verifier.js:32
2019-08-19T22:31:14.740517410Z     return mandatoryVariables.filter((variable) => {
2019-08-19T22:31:14.740522027Z                               ^
2019-08-19T22:31:14.740526124Z 
2019-08-19T22:31:14.740528910Z TypeError: Cannot read property 'filter' of undefined
2019-08-19T22:31:14.740531511Z     at getMissingVariables (/usr/app/utils/verifier.js:32:31)
2019-08-19T22:31:14.740534109Z     at Object.module.exports.verifyEnvironmentVars (/usr/app/utils/verifier.js:20:35)
2019-08-19T22:31:14.740537628Z     at Object.<anonymous> (/usr/app/app.js:3:29)
2019-08-19T22:31:14.740540400Z     at Module._compile (module.js:653:30)
2019-08-19T22:31:14.740542985Z     at Object.Module._extensions..js (module.js:664:10)
2019-08-19T22:31:14.740545576Z     at Module.load (module.js:566:32)
2019-08-19T22:31:14.740548046Z     at tryModuleLoad (module.js:506:12)
2019-08-19T22:31:14.740550536Z     at Function.Module._load (module.js:498:3)
2019-08-19T22:31:14.740553029Z     at Function.Module.runMain (module.js:694:10)
2019-08-19T22:31:14.740555547Z     at startup (bootstrap_node.js:204:16)
```

**To Reproduce**
1. Setup influx as metric plugin and the necessary fields to point to your influxdb
2. Create a test and run it

**Expected behavior**
Test should run, data should be sent to influxdb

**Screenshots**
None

**Version:**
 - Predator: latest
 - Predator-runner latest
 - Database built-in

**Additional context**
No context specific

From my understanding of https://github.com/Zooz/predator-runner/blob/master/app/utils/consts.js#L12 , the correct value should be 'influxdb' instead of 'influx' for the metrics plugin name, and then all the properties should have a "influxdb_" prefix.

**Comment by NivLipetz** [comment]
Hi @tyge68 thanks for reporting the bug! We are working on a release

**Comment by enudler** [comment]
The bug is fixed in predator-runner 1.1.7:
https://github.com/Zooz/predator-runner/blob/master/CHANGELOG.md

Thanks to @NivLipetz
